salt/_pillar/metalk8s.py

```diff
@@ -70,6 +71,7 @@ def _load_networks(config_data):
         'workload_plane': networks_data['workloadPlane'],
         'pod': networks_data.get('pods', DEFAULT_POD_NETWORK),
         'service': networks_data.get('services', DEFAULT_SERVICE_NETWORK),
+        'mtu': networks_data.get('mtu', DEFAULT_MTU)
```

Comment:
I don't think this is at the right 'location' in the config. It's not something *next to* the pod, service and other network CIDRs. It is *the MTU of the workload plane network*, so it should go in there. Indeed, that means bumping the version of the config format.

Reply:
Right, as discussed, I will update it